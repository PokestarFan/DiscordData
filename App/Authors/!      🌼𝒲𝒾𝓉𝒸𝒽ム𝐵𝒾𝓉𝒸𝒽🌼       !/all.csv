ID,Server,Channel,Content Length,Content
558465768443084801,FlareTV - PewDiePie vs T-Series,general,3,uwu
558466578048483330,FlareTV - PewDiePie vs T-Series,general,9,K P O P ?
558466631223738377,FlareTV - PewDiePie vs T-Series,general,20,I have been summoned
558466708244004892,FlareTV - PewDiePie vs T-Series,general,7,Damn it
558466820517003266,FlareTV - PewDiePie vs T-Series,general,41,*crawls back in the hellhole I came from*
558466868529070090,FlareTV - PewDiePie vs T-Series,general,3,uwu
558467003334131749,FlareTV - PewDiePie vs T-Series,general,17,I already do that
558467162398785566,FlareTV - PewDiePie vs T-Series,general,9,I am hell
558467342191820800,FlareTV - PewDiePie vs T-Series,general,35,<@558457743761997825> no_ I AM hell
558467663756525589,FlareTV - PewDiePie vs T-Series,general,2,Ew
558467733851865098,FlareTV - PewDiePie vs T-Series,general,28,Thats nastier than egg salad
558467790739079179,FlareTV - PewDiePie vs T-Series,general,4,Wait
558467948969197589,FlareTV - PewDiePie vs T-Series,general,60,Why do we have so many people named after food in this sever
558468046264729640,FlareTV - PewDiePie vs T-Series,general,8,Probably
558468092653469697,FlareTV - PewDiePie vs T-Series,general,4,Dog?
558468118738108417,FlareTV - PewDiePie vs T-Series,general,14,Dog isn’t food
558468196202577920,FlareTV - PewDiePie vs T-Series,general,19,Crab is also a good
558468258018230283,FlareTV - PewDiePie vs T-Series,general,17,Food_ I mean food
558468323696836620,FlareTV - PewDiePie vs T-Series,general,27,<@!380928993144340482> SAME
558468516924227595,FlareTV - PewDiePie vs T-Series,general,50,<@!380928993144340482> I’ve never had crAwfish ;-;
558468576000868382,FlareTV - PewDiePie vs T-Series,general,17,Hi bored_ I’m dad
558468680195768320,FlareTV - PewDiePie vs T-Series,general,37,<@!380928993144340482> I just havrnts
558468757333213211,FlareTV - PewDiePie vs T-Series,general,16,Stop flexing aaa
558468805743869998,FlareTV - PewDiePie vs T-Series,general,39,<@!380928993144340482> I will then lmao
558469005073973268,FlareTV - PewDiePie vs T-Series,general,39,<@!380928993144340482> I don’t think so
558469086317772830,FlareTV - PewDiePie vs T-Series,general,30,*crawls out of hellhole* kPoP?
558469157608489020,FlareTV - PewDiePie vs T-Series,general,27,<@558457743761997825> what?
558469381953159168,FlareTV - PewDiePie vs T-Series,general,32,<@!380928993144340482> oooh okay
558469529177686017,FlareTV - PewDiePie vs T-Series,general,34,Imma block eclipse if Chelsea does
558469548789989386,FlareTV - PewDiePie vs T-Series,general,6,Oh rip
558469642855907328,FlareTV - PewDiePie vs T-Series,general,34,Damn I needed a reason to be petty
558469747008733194,FlareTV - PewDiePie vs T-Series,general,4,Yeah
558469812834140181,FlareTV - PewDiePie vs T-Series,general,3,Ban
558470043776581632,FlareTV - PewDiePie vs T-Series,general,3,Rip
558470153851895839,FlareTV - PewDiePie vs T-Series,general,3,Yes
558470374392725517,FlareTV - PewDiePie vs T-Series,general,11,Hit or miss
558470421343633408,FlareTV - PewDiePie vs T-Series,general,27,I guess they never miss huh
558470443762450444,FlareTV - PewDiePie vs T-Series,general,19,You got a boyfriend
558470476377096200,FlareTV - PewDiePie vs T-Series,general,24,I bet he doesn’t kiss ya
558470519138287617,FlareTV - PewDiePie vs T-Series,general,4,Muga
558470548598816778,FlareTV - PewDiePie vs T-Series,general,7,PRIVATE
558470559483035668,FlareTV - PewDiePie vs T-Series,general,7,YOU HOE
558470610825510922,FlareTV - PewDiePie vs T-Series,general,32,Stop making me sound like a thot
558470708687011848,FlareTV - PewDiePie vs T-Series,general,13,*no it’s not*
558470742623125517,FlareTV - PewDiePie vs T-Series,general,2,NO
558470758737641485,FlareTV - PewDiePie vs T-Series,general,4,Lmao
558470858683711488,FlareTV - PewDiePie vs T-Series,general,25,Excuse me_ I’m a guy lmao
558471124900380693,FlareTV - PewDiePie vs T-Series,general,17,Egg is not a girl
558471316299186186,FlareTV - PewDiePie vs T-Series,general,2,he
558471329993719819,FlareTV - PewDiePie vs T-Series,general,8,I’m a he
558471663167995914,FlareTV - PewDiePie vs T-Series,general,33,<@!380928993144340482> crab is 10
558472219748204548,FlareTV - PewDiePie vs T-Series,general,21,There’s a 10 year old
558473347428515841,FlareTV - PewDiePie vs T-Series,general,0,
558778138558070795,FlareTV - PewDiePie vs T-Series,general,4,Shit
558778153582198784,FlareTV - PewDiePie vs T-Series,general,4,Fucc
558778191238660096,FlareTV - PewDiePie vs T-Series,general,5,Bitsh
558778226692849664,FlareTV - PewDiePie vs T-Series,general,3,Hoe
558798908881633290,FlareTV - PewDiePie vs T-Series,general,3,uwu
558810331376386072,FlareTV - PewDiePie vs T-Series,general,20,Aaaa I wanna join vc
558810365690118167,FlareTV - PewDiePie vs T-Series,general,21,But I’m in another rn
558810390570729483,FlareTV - PewDiePie vs T-Series,general,14,I can’t choose
558811900645736468,FlareTV - PewDiePie vs T-Series,general,15,can i be banned
558811907855745024,FlareTV - PewDiePie vs T-Series,general,3,pls
558812046381154305,FlareTV - PewDiePie vs T-Series,general,6,i cant
558813039030304799,FlareTV - PewDiePie vs T-Series,general,0,
558813202545377312,FlareTV - PewDiePie vs T-Series,general,22,<@!176921245940580354>
558814011878014976,FlareTV - PewDiePie vs T-Series,general,73,my milkshake brings all the boys to yard_ damn right is better than yours
558815672562483210,FlareTV - PewDiePie vs T-Series,general,7,oh okay
558817008083402763,FlareTV - PewDiePie vs T-Series,general,3,hoe
558818391738286121,FlareTV - PewDiePie vs T-Series,general,25,family friendly porn_ wtf
558818749264691268,FlareTV - PewDiePie vs T-Series,general,11,move me too
558818784387923974,FlareTV - PewDiePie vs T-Series,general,20,wow i didnt even ask
558991746395013121,FlareTV - PewDiePie vs T-Series,general,3,Wot
558992002637627393,FlareTV - PewDiePie vs T-Series,general,2,Oh
558992235622825994,FlareTV - PewDiePie vs T-Series,general,7,Oh nice
558992407476043795,FlareTV - PewDiePie vs T-Series,general,3,oof
558992649881911316,FlareTV - PewDiePie vs T-Series,general,20,Nothing personal kid
558992830853414964,FlareTV - PewDiePie vs T-Series,general,19,Kid is not a virgin
558992953888997399,FlareTV - PewDiePie vs T-Series,general,2,Oh
558993192406482954,FlareTV - PewDiePie vs T-Series,general,8,You suck
558993253920276480,FlareTV - PewDiePie vs T-Series,general,34,Morning Bart <@339845222181896193>
558993353266429973,FlareTV - PewDiePie vs T-Series,general,26,<@!285835383374348289> oOf
558993702765461514,FlareTV - PewDiePie vs T-Series,commands,5,!rank
558996094672175104,FlareTV - PewDiePie vs T-Series,general,3,Oof
558996235776950283,FlareTV - PewDiePie vs T-Series,general,5,Dunno
