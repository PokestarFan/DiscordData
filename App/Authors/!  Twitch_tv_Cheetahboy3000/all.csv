ID,Server,Channel,Content Length,Content
560642432505610260,Official Fortnite,fortnite-general,18,any youtubers here
560642582359834635,Official Fortnite,stream-video-promo,18,any youtubers here
560642909934977034,Official Fortnite,fortnite-general,31,this game is dead why am i here
