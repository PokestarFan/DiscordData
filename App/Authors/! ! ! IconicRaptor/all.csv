ID,Server,Channel,Content Length,Content
558444909892468739,MINECRAFT,minecraft,3,wew
558445197797752833,MINECRAFT,minecraft,3,wew
558445222368116756,MINECRAFT,minecraft,8,lmao so?
558449266033688591,MINECRAFT,minecraft,3,wew
558449490676416512,MINECRAFT,minecraft,38,discussion related to pirated content?
558449517368967210,MINECRAFT,minecraft,5,hmmmm
558820312519540739,MINECRAFT,minecraft,3,Wew
558820337060675616,MINECRAFT,minecraft,37,minecraft central is so laggy rn lmao
558820365430947840,MINECRAFT,minecraft,28,IM TRYING TO GRIND SUGARCANE
558821594957611028,MINECRAFT,minecraft,4,wew\
558823421387997213,MINECRAFT,minecraft,4,\wew
558823862024929281,MINECRAFT,minecraft,64,<@!484506513881169940> jokes on you tnt is the most useful block
558823904563691520,MINECRAFT,minecraft,4,yeah
558823933441212416,MINECRAFT,minecraft,20,all splash pots lmao
558823962067468288,MINECRAFT,minecraft,23,ew nooo fireworks greay
558824391912194049,MINECRAFT,minecraft,54,<@515846663496138772> who hasnt made a tnt cannon lmao
558824436908687380,MINECRAFT,minecraft,35,Everyone plays factions Im assuming
558824480021938206,MINECRAFT,minecraft,4,lool
558824509658890250,MINECRAFT,minecraft,47,2000 blocks is only 125 chunks which is nothing
558824572380512296,MINECRAFT,minecraft,2,:o
558824601656885256,MINECRAFT,minecraft,41,you dont raid people on smp? what noob xD
558824682669735940,MINECRAFT,minecraft,36,Lol you mean ur going to /thor them?
558824705981939732,MINECRAFT,minecraft,11,lmao ok bud
558824920897945628,MINECRAFT,minecraft,26,<@!484506513881169940> wew
558824987398504467,MINECRAFT,minecraft,52,Remember when people were actually good at factions?
558825014913400832,MINECRAFT,minecraft,28,Yeah that was 2 years ago :l
558825116511895554,MINECRAFT,minecraft,3,lol
558825146358693899,MINECRAFT,minecraft,38,Or you could make a 500 chunk tunneler
558825969998364693,MINECRAFT,minecraft,3,wew
