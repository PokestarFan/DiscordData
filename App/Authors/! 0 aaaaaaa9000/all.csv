ID,Server,Channel,Content Length,Content
558792322213609482,Official Fortnite,stw-trade,47,[H] Malachite [W] Lots and lots of brick [P] PC
