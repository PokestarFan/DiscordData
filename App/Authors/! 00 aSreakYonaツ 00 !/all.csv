ID,Server,Channel,Content Length,Content
559031027239223296,Official Fortnite,fortnite-general,26,anyone in asia down to 1v1
