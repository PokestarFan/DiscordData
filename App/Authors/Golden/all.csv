ID,Server,Channel,Content Length,Content
560653879788830732,FlareTV - PewDiePie vs T-Series,general,7,Remebet
560653914249101332,FlareTV - PewDiePie vs T-Series,general,10,We can win
560654206491688980,FlareTV - PewDiePie vs T-Series,pewdiepie-vs-tseries,32,They are fucking 1000 subs ahead
560654256739450900,FlareTV - PewDiePie vs T-Series,pewdiepie-vs-tseries,7,And how
560654611191431210,FlareTV - PewDiePie vs T-Series,pewdiepie-vs-tseries,22,Join public lobby vc 2
